api/controllers/case.js

```diff
@@ -196,11 +196,11 @@ function getUpdatedCase(user, params, newCase, oldCase) {
   }
 
   // media lists
-  ["links", "videos", "audio", "evaluation_links"].map(
-    key => cond(key, as.media)
-  );
+  ["links", "videos", "audio"].map(key => cond(key, as.media));
   // photos and files are slightly different from other media as they have a source url too
-  ["photos", "files", "evaluation_reports"].map(key => cond(key, as.sourcedMedia));
+  ["photos", "files", "evaluation_reports", "evaluation_links"].map(key =>
```

Comment:
`evaluation_links` should be like links, videos, and audio. it does not handle uploaded media.

Reply:
You're right, of course. I managed to mis-read the spec again.